skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkiaLayer.ios.kt

```diff
@@ -185,7 +185,8 @@ actual open class SkiaLayer {
 }
 
 // TODO: do properly
+actual typealias SkikoTouchPlatformEvent = UITouch
 actual typealias SkikoGesturePlatformEvent = UIEvent
-actual typealias SkikoPlatformInputEvent = UIEvent
+actual typealias SkikoPlatformInputEvent = UIPress
```

Comment:
Why `UIPress`?

Reply:
It is because on ios there is no `uievent` when you type, only methods like `override fun insertText(theText: String) {`, but instead of that we also need to pass key event (`uipress`), so I decided to set `SkikoPlatformInputEvent` as `UIPress` alias